### DISCUSSION TITLE
versions.somepythonthings.tk

### DISCUSSION THREAD

**Discussion by Chris-in-Makati** [martinet101/ElevenClock#1323] [General]
Over the last couple of days Malwarebytes has been notifying me that it has blocked a connection attempt by ElevenClock to "versions.somepythonthings.tk". I've researched that URL and it does appear to be suspicious. Why is ElevenClock connecting to that site, and should it be doing so?

**Comment by marticliment** [comment]
Hello,

This is an old domain of mine that got lost. Newer versions of ElevenClock have those domains updated to the new one, but older versions didn't catch the update (I suppose you sre using ElevenClock 3.9.6). Please install ElevenClock from the microsoft store and the app will update itself.

**Reply by Chris-in-Makati** [reply]
Thanks for that.

I was on 3.9.6 which I had previously downloaded from GitHub. I didn't know the app was now distributed on the Microsoft Store.

So I've now uninstalled the previous version and reinstalled from the MS Store.

All good now, and thanks for all your hard work making such a great app available.
